charts/idam-user-dashboard/values.yaml

```diff
@@ -23,6 +23,7 @@ nodejs:
     STRATEGIC_PUBLIC_URL: 'https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net'
     DASHBOARD_URL: 'https://idam-user-dashboard.{{ .Values.global.environment }}.platform.hmcts.net'
     OAUTH_CLIENT_REDIRECT: 'https://idam-user-dashboard.{{ .Values.global.environment }}.platform.hmcts.net/oauth2/callback'
+    TESTING_SUPPORT_URL: 'https://idam-testing-support-api.{{ .Values.global.environment }}.platform.hmcts.net'
```

Comment:
was it hardcoded somewhere before? Can't see how this gets referenced in this PR (or replaces whatever was already there)

Reply:
yeah its hardcoded in defaults.yaml

Reply:
Shouldn't the values in [default.yaml](https://github.com/hmcts/idam-user-dashboard/blob/master/config/default.yaml)  be  changed to?

Reply:
they are the defaults, what should I change them to?

Reply:
I think tests use the defaults, don't they? I thought that's why we can't run them against non-aat envs?

Reply:
custom-environment-variables.yaml is supposed to overwrite the defaults.yaml
They do if I try it locally so I am going to assume that its going to work in the pipeline.
I didn't add the variable to the nightly pipeline though, I did now. I forgot to do that

Reply:
Yeah, the tests get it like this ``${config.get('services.idam.url.testingSupportApi')}/test/idam/users`,` and they run in the pipeline, so would the override `testingSupportApi: TESTING_SUPPORT_URL` have a value when they run?

Reply:
oh is this for master too? okay I will add it to the main pipeline I guess

Reply:
I mean, you need it in all the jenkins files don't you?

Reply:
I assumed this was only for running the nightly pipeline, since as far as I know the functionals wont run on other environments, but I will add it to the master one if everyone wants that

Reply:
ah right, I guess dashboard doesn't use preview at all, so you're probably right. I'm confused about the environment overrides though - do they just get ignored if they're empty and the default value is picked up?

Reply:
yes, if the env variable is empty the default is used

Reply:
I think I'm confused - I didn't get why we need testingSupportApi in the helm chart?